docs/DeveloperGuide.md

```diff
@@ -72,28 +88,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Pairs Volunteer and Elderly**
```

Comment:
For the use cases, isn't @heeeyi suppose to fill it in? (At least according to the minutes). I'm fine either way.

Reply:
I have done this part for all the Use cases :)

Reply:
Alright, nice!